src/schema/nmdc.yaml

```diff
@@ -382,7 +382,11 @@ classes:
         range: AnnotatingWorkflow
       count: 
           description: The number of sequences (for a metagenome or metatranscriptome) or spectra (for metaproteomics) associated with the specified function.
-
+      gene_function_id:
+        structured_pattern:
+          syntax: "(COG:COG\\d+|PFAM:PF\\d{5}|KEGG.ORTHOLOGY:K\\d+)"
```

Comment:
My hunch is that this is appropriate

But if you aren't using any of the  shema's`settings` in the `syntax`, then you might as well make this a `pattern`, not a `structured_pattern`. If you were using `settings`, they would appear as symbols inside of `{}`s in the `syntax`.  I'm not sure how that combines with character count constraints like `{5}`

If that doesn't' make sense, then we can discuss in whatever channels you prefer